tests/baseSetup.go

```diff
@@ -36,6 +37,9 @@ var (
 //One init function per package - initializes Rook infra and installs rook(if needed based on flags)
 func init() {
 	Env = objects.NewManifest()
+	if kv := os.Getenv("KUBE_VERSION"); kv != "" {
```

Comment:
I am going to clean this up as part of #774  the flags names have to be consistent to use and Flasg.setwithEnv and clean up unused flags.